pkg/cluster/delete.go

```diff
@@ -63,3 +90,57 @@ func Delete(
 	}
 	return nil
 }
+
+func DeleteOverrideDir(appIf argoapp.ApplicationServiceClient, kubeClient *kubernetes.Clientset, argocdNs string, name string) error {
+	log := logpkg.GetLogger()
+	app, err := appIf.Get(context.Background(),
```

Comment:
This function should not need to get the app again. The caller already has a copy of the app. Pass it by pointer.

Reply:
Passed the app pointer instead of appIf